### DISCUSSION TITLE
Capturing Firefly trails

### DISCUSSION THREAD

**Discussion by mburge** [aaronwmorris/indi-allsky#1277] [Show and tell]
It's firefly season in North America. With the stacking option set to maximum and a tilt towards the horizon the star trails composite shows hundreds of paths. Zoom in around the trees to see. Also a comet up in the north quadrant. Thanks again Aaron for a wonderful software package. 

![image](https://github.com/aaronwmorris/indi-allsky/assets/3086001/c6beec7f-eb27-41e4-b89f-329ba9dac468)

Detail with contrast and exposure bumped:

![image](https://github.com/aaronwmorris/indi-allsky/assets/3086001/db2f948f-451d-4efe-bbde-e9fbb917ef4a)

**Comment by aaronwmorris** [comment]
That is absolutely amazing!
